src/main/java/net/barrage/school/java/ecatalog/config/SecurityConfiguration.java

```diff
@@ -72,4 +84,15 @@ public JwtAuthenticationConverter jwtAuthenticationConverter() {
         );
         return converter;
     }
+
+    @Bean
+    @Transactional
```

Comment:
why?

Reply:
Woops, accidentally left it. Im deleting it